mlflow/keras_core/__init__.py

```diff
@@ -1,3 +0,0 @@
-from mlflow.keras_core.callback import MLflowCallback
-
-__all__ = ["MLflowCallback"]
```

Comment:
should we raise a deprecation warning?

Reply:
it was marked as experimental, and I don't think anyone is using it, so I guess we can just delete it.